send-emails-from-a-service/README.md

```diff
@@ -8,7 +8,7 @@ To implement this use case, you will develop a REST service with a single resour
 
 The flow is as follows.
 
-1. Receive a request with a JSON payload similar the following form.
+1. Receive a request with a JSON payload similar to the following form.
```

Comment:
```suggestion
1. Receive a request with a JSON payload in the following form.
```

Since it is the structure we are talking about, doesn't need similar IMO.